### DISCUSSION TITLE
Deep Image Prior for Inverse Problems in Imaging (Project 244)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#80] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project 244](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/blob/main/projects/Deep%20Image%20Prior%20for%20Inverse%20Problems%20in%20Imaging)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by ghost** [comment]
Hi, This is Sasi, a student at IIT Tirupati, interested in collaborating and contributing to this project. :-)

**Comment by SasiAlavala** [comment]
Hi, This is Sasi, a student at IIT Tirupati, interested in collaborating and contributing to this project. :-)

**Reply by cps13** [reply]
Hi Sasi, thanks for showing interest in this project. If you get stuck, feel free to reply here and I can see if I can help you out with it 😊 Good luck!

**Comment by maneetchatterjee** [comment]
Hi ,this is Maneet, interested to join this project and collaborate with you.

**Comment by razarschX** [comment]
Hi, my name is Raza, currently a undergraduate student at Abdullah Gul University in Turkey; looking forward to working and collaborating on this project

**Comment by mshrihari3** [comment]
Hi, my name is Shrihari, and I was wondering about "Advanced Extensions", to get better results, are we allowed to use different techniques like DCGAN or methods other than Deep Prior?

**Comment by Coding-Turtle0** [comment]
Hi, my name is Sagnik Chakraborty, a student of Institute of Engineering & Management, Kolkata and interested in working and collaborating on this project.

**Comment by leezongzhe** [comment]
Hi, my name is Lee, a student at Universiti Malaya, department of Biomedical Engineering and interested in working and collaborating on this project

**Comment by AdarshKorde** [comment]
Hi, I am a student at IIT Bombay doing specialization in systems and control engineering and interested in working and collaborating on this project.
